lib/pdf/core/sRGB2014.icc.license

```diff
@@ -0,0 +1,10 @@
+ICC sRGB v2 Profile
```

Comment:
The profile is not code. Please put it in the `data` directory and please don't forget to add it to the packaged gem as well.

Also as far as I can tell copyright information and profile name is included in the profile itself. The license doesn't seem to require this file distribution along with the profile so maybe drop it. Include the links in commit message for history. That should be enough.